packages/go_router_builder/lib/src/route_config.dart

```diff
@@ -412,6 +412,8 @@ bool $boolConverterHelperName(String value) {
 
 const String _enumConverterHelper = '''
 extension<T extends Enum> on Map<T, String> {
-  T $enumExtensionHelperName(String value) =>
-      entries.singleWhere((element) => element.value == value).key;
+  T $enumExtensionHelperName(String value) => entries
+      .singleWhere(
+          (element) => element.value.toLowerCase() == value.toLowerCase())
```

Comment:
This will cause issue if you have enum like this
```dart
enum ABC{
  aaa,
  aAA,
  AaA,
}
```

I think we should not hardcode this behavior

Reply:
Good point, I think the https://github.com/flutter/flutter/issues/110781 proposal is much better, closing this one in favor and might open a PR for 110781 if I get the time, thank you!